### DISCUSSION TITLE
How to convert a date string to a date?

### DISCUSSION THREAD

**Discussion by ghost** [js-fns/date-fns#2129] [Q&A] [answered]
I want to pass a string that's formatted as a date (e.g. '10/21/2020') and return a DATE object.

I thought toDate would do it, but it seems to only accept dates and numbers.

I thought Parse might do it, but I don't understand the documentation.

It seems this should  be a common need.

How can I do it?

**Comment by efenderbosch** [comment, marked as answer]
`parse('10/21/2020', 'dd-MM-yyyy', 0)`
